tests/Tests/500__jupyterhub/test.robot

```diff
@@ -74,7 +75,26 @@ Can Launch Python3
    [Tags]  Sanity  TBC
    Launch Python3 JupyterHub
 
+Verify Failing Images Are Shown In Notifications
+    [Documentation]    Verifies failing Notebook names are shown in RHODS Dashboard Notifications
+    [Tags]    Sanity
+    ...       ODS-470
+    ${failed_build_name} =    Provoke Image Build Failure    namespace=redhat-ods-applications
+    ...    build_name_includes=pytorch    build_config_name=s2i-pytorch-gpu-cuda-11.4.2-notebook
+    ...    container_to_kill=sti-build
+    Verify Failed Build In Notifications
+    [Teardown]    Delete Failed Build And Start New One   namespace=redhat-ods-applications  failed_build_name=${failed_build_name}  build_config_name=s2i-pytorch-gpu-cuda-11.4.2-notebook
+
 
 *** Keywords ***
 JupyterHub Testing Suite Setup
   Set Library Search Order  SeleniumLibrary
+
+Verify Failed Build In Notifications
+    [Documentation]    Verifies Notifications show failed Notebook name
+    Open Browser    ${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}    options=${BROWSER.OPTIONS}
```

Comment:
you can use `Launch Dashboard` keyword instead of repeating these 3 lines

Reply:
Updated